README.md

```diff
@@ -26,33 +27,59 @@ Alternativly, simply add these lines to your /etc/hosts:
 ::1 ci.foo.redhat.com
 ```
 
-## Getting started
+## Getting started - Installation
 
 1. Make sure [nvm](https://github.com/nvm-sh/nvm) is installed
 
-2. First time running the app do: `nvm use` to ensure you have the correct node version installed. If you do not, follow the instructions nvm gives you to install the appropriate version.
+2. First time running the app do: `nvm use` to ensure you have the correct node version installed.
+   If you do not, follow the instructions nvm gives you to install the appropriate version.
+   Yarn WILL prevent you from progressing if you have not updated your node version.
+
+3. `yarn install`
+
+4. `yarn build` (only required when setting up for the first time)
 
-3. `npm install`
+## Getting started - Running the app
 
-4. `npm run build` (only required when setting up for the first time)
+Keep in mind that you have to be connected to the VPN for this to work, even in the offices.
 
-5. `npm run start` to choose whether to run against stage or prod environments. <br/>
+1. `yarn start` to choose whether to run against stage or prod environments. <br/>
    OR <br/>
-   `npm run start:stage` to run against stage environment. <br/>
+   `yarn start:stage` to run against stage environment. <br/>
    OR <br/>
-   `npm run start:prod` to run against prod environment. <br/>
+   `yarn start:prod` to run against prod environment. <br/>
    OR <br/>
-   `npm run local` to run against a local backend running on port 8000. <br/>
-   (keep in mind that you have to be connected to the VPN for this to work, even in the offices)
+   `yarn local` to run against our local backend running on port 8000.<br/>
+
+2. With a browser, open the URL listed in the terminal output, https://stage.foo.redhat.com:1337/insights/content for example.
+
+### Unit Testing
+
+`yarn verify` will run `yarn build` `yarn lint` (eslint), `yarn format:check` Prettier formatting check and `yarn test` (Jest unit tests)
+
+Alternatively one can run simply: `yarn test` to run the unit tests.
```

Comment:
```suggestion
Run `yarn test` to run the unit tests.
```"simply" is marketing speak. In technical writing, never make judgements about how hard or easy something is.